### ISSUE TITLE
Bug | Rubocop 0.89 | uninitialized constant RuboCop::Cop::TooManyLines

### ISSUE THREAD

**Issue by dzemlianoi-double** [DmitryTsepelev/rubocop-graphql#23]
Hi guys, i have met the issue with updating of base rubocop version from 0.88 -> 0.89

File with module RuboCop::Cop::TooManyLines has been removed in [this commit](https://github.com/rubocop-hq/rubocop/commit/5b7c51c533341905b1ea12c85ae78a1591eb2c0d)

All the places with this constant usage have been replaced with Rubocop::Cop::CodeLength.
Also module `CodeLength` has been patched to handle both of cases - previous behaviour of CodeLength module and TooManyLines functionality

## Expected behavior

RuboCop is running successfully

## Actual behavior

Error is raised during rubocop starting
```
uninitialized constant RuboCop::Cop::TooManyLines
/Users/my_user/.rvm/gems/ruby-2.6.5@my_project/gems/rubocop-graphql-0.4.0/lib/rubocop/cop/graphql/resolver_method_length.rb:12:in `<class:ResolverMethodLength>'
```

## Steps to reproduce the problem

Run the rubocop v0.89.0 with included rubocop-graphql gem

## RuboCop version
```
0.89.0 (using Parser 2.5.1.2, running on ruby 2.6.5 x86_64-linux)
```

## Proposed steps to fix

1) Bump up the version of Rubocop-Graphql (major(preferred) / minor), set required version of rubocop greater or equal than 0.89 and rename the module included to `resolver_method_length` Cop
2) Make the condition inside the `resolver_method_length` Cop with check the version of rubocop and including correct constant(this check could be done at the start of rubocop running - not in this file)

**Comment by DmitryTsepelev** [comment]
Hi @dzemlianoi-double, thanks for the report! I'll take a look at the issue when my vacation ends 🏖

**Comment by DmitryTsepelev** [comment]
Fixed as part of PR #24 and released a new version
